ocs_sample_library_preview/Securable.py

```diff
@@ -0,0 +1,210 @@
+from ocs_sample_library_preview.Security.AccessRights import AccessRights
```

Comment:
Should just be `from .Security.AccessRights` I think? We don't usually use the full library name for imports within the same package. This should also be moved to the second section of imports, which all come from within the package.

Reply:
Oops. I let that one be autogenerated. I changed it to ```from .Security.AccessRights```